lib/movie_card/movie_actions_play.dart

```diff
@@ -0,0 +1,27 @@
+part of 'movie_actions.dart';
+
+class _PlayTrailerButton extends StatelessWidget {
```

Comment:
there is no need to make it part of the other widget, keep it as a new widget, remove the underscore from the name and make the file name similar as the class name